packages/fast-components-styles-msft/src/radio/index.ts

```diff
@@ -102,13 +112,35 @@ const styles: ComponentStyles<RadioClassNameContract, DesignSystem> = {
         ...applyScaledTypeRamp("t7"),
         marginLeft: directionSwitch(horizontalSpacing(2), ""),
         marginRight: directionSwitch("", horizontalSpacing(2)),
+        [highContrastSelector]: {
+            color: "ButtonText",
+        },
     },
     radio__checked: {
         "& $radio_stateIndicator": {
             "&::before": {
                 background: neutralForegroundRest,
-                "@media (-ms-high-contrast:active)": {
-                    backgroundColor: "ButtonHighlight",
+                [highContrastSelector]: {
+                    background: "Highlight",
+                },
+            },
+        },
+        "&:hover": {
+            "& $radio_stateIndicator": {
+                "&::before": {
```

Comment:
instead of nesting things like this you con combine them to be `"&:hover $radio_stateIndicator::before"`

Reply:
combined